bezier/src/cubic_bezier.rs

```diff
@@ -397,6 +483,25 @@ fn find_y_minimum_for_simple_cubic_segment() {
 }
 
 #[test]
+fn find_y_extremums_for_simple_cubic_segment() {
+    let a = CubicBezierSegment {
+        from: Point::new(1.0, 1.0),
+        ctrl1: Point::new(1.0, 2.0),
+        ctrl2: Point::new(2.0, 1.0),
+        to: Point::new(2.0, 2.0),
+    };
+
+    let mut expected_y_extremums = UpToTwo::new();
+    expected_y_extremums.push(0.5);
```

Comment:
I don't think that 0.5 is an `y-inflection` point of the curve (although it is an inflection point). After drawing it on a piece of paper it looks like this curve is monotone (always increasing) along y.
I think that this curve should have no `y-inflections` (and also no `x-inflections`), even if, confusingly, 0.5 is a "normal" inflection point.